apph-front/src/static/components/CreateFolderButton.tsx

```diff
@@ -24,24 +24,23 @@ export const CreateFolderButton = ({
   const [errorMessage, setErrorMessage] = useState('');
   const [loading, setLoading] = useState<boolean>(false);
 
-  const handleSubmit = () => {
+  const handleSubmit = async () => {
     if (folderName === '') {
       setErrorMessage('Le nom du dossier ne peut pas être vide.');
     } else {
       setLoading(true);
-      FolderService.createFolder(
+      await FolderService.createFolder(
```

Comment:
pour rester 
```suggestion
      FolderService.createFolder(
```

pourquoi avoir utilisé await ici ? pour rester cohérent avec les autres composants il faudrait l'enlever, nos méthodes ne sont jamais asynchrones dans les composants
l'ancienne version avec setLoading(false) dans les callbacks fonctionnait non ?

Reply:
c'est pour enlever le warning moche, on ne le fait pas dans useEffect psk il ne peut pas être asynchrone

Reply:
et ça fait une ligne de setLoading(false) en moins